weave/scorers/guardrails/presidio_entity_recognition_guardrail.py

```diff
@@ -0,0 +1,205 @@
+from typing import TYPE_CHECKING, Any, Optional
+
+from pydantic import BaseModel
+
+import weave
+from weave.scorers import Scorer
+
+if TYPE_CHECKING:
+    from presidio_analyzer import AnalyzerEngine, RecognizerResult
+    from presidio_anonymizer import AnonymizerEngine
+
+
+class PresidioEntityRecognitionResponse(BaseModel):
```

Comment:
@tcapelle @soumik12345 are you going with TypedDict or BaseModel?  Either is fine, but I would pick 1 and be consistent for all the scorers.  Maybe you should also have a test to enforce this property

Reply:
I checked the hallucination and summarization scorers, it seems like we're consistently using `BaseModel`.